packages/web-components/project.json

```
@@ -3,5 +3,10 @@
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "projectType": "library",
   "implicitDependencies": [],
+  "targets": {
+    "e2e": {
+      "dependsOn": [{ "target": "build", "projects": "dependencies" }]
+    }
+  },
```

no need to use the verbose version

```suggestion
  "targets": {
    "e2e": {
      "dependsOn": ["build"]
    }
  },
```

now do you need to physically build web-component assets as well ( I guess not) ? Then you can do ` "dependsOn": ["^build"]` which will build only dependants = faster CI.


 I thought your vite setup does the processing via path aliases like rest of monorepo packages so everything is build with esbuild behind the scenes ( which is significantly faster than triggering dependants build targets )